docs/integrations/sources/instagram.md

```diff
@@ -0,0 +1,74 @@
+# Instagram
+
+## Sync overview
+
+This source can sync data for the Instagram Business Account available in the Facebook Graph API: User, Media, and Stories. It can also sync Media/Story and User Insights.
+
+### Output schema
+
+This Source is capable of syncing the following core Streams:
+
+* [User](https://developers.facebook.com/docs/instagram-api/reference/ig-user)
+  * [User Insights](https://developers.facebook.com/docs/instagram-api/reference/ig-user/insights)
+* [Media](https://developers.facebook.com/docs/instagram-api/reference/ig-user/media)
+  * [Media Insights](https://developers.facebook.com/docs/instagram-api/reference/ig-media/insights)
+* [Stories](https://developers.facebook.com/docs/instagram-api/reference/ig-user/stories/)
+  * [Story Insights](https://developers.facebook.com/docs/instagram-api/reference/ig-media/insights)
+
+For more information, see the [Instagram API ](https://developers.facebook.com/docs/instagram-api/) and [Instagram Insights API documentation](https://developers.facebook.com/docs/instagram-api/guides/insights/).
+
+### Data type mapping
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
+| `string` | `string` |  |
+| `number` | `number` |  |
+| `array` | `array` |  |
+| `object` | `object` |  |
+
+### Features
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental Sync | Yes | only User Insights |
+
+### Rate Limiting & Performance Considerations
+
+Instagram, like all Facebook services, has a limit on the number of requests.
+
+See Facebook's [documentation on rate limiting](https://developers.facebook.com/docs/graph-api/overview/rate-limiting/#instagram-graph-api) for more information.
+
+## Getting started
+
+### Requirements
+
+* A Facebook App 
+* An Instagram Business Account
+* A Facebook API Access Token
+
+### Setup guide
+
+### Facebook App
+
+#### If you don't have a Facebook App
+
+Visit the [Facebook Developers App hub](https://developers.facebook.com/apps/) and create an App and choose "Manage Business Integrations" as the purpose of the app. Fill out the remaining fields to create your app.
+
+### Instagram Business Account
+
+Follow the [Instagram documentation](https://www.facebook.com/business/help/1492627900875762) for setting up an Instagram business account. We'll need this ID to configure Instagram as a source in Airbyte.
+
+### API Access Token
+
+To work with the Instagram connector, you need to generate an Access Token with the following permissions:
+
+* [instagram_basic](https://developers.facebook.com/docs/permissions/reference#instagram-basic)
+* [instagram_content_publish](https://developers.facebook.com/docs/permissions/reference#instagram-content-publish)
```

Comment:
why do we need `content_publish`? we are not publishing anything.

Reply:
This permission is really not needed, removed from docs